Mathlib/Algebra/Algebra/NonUnitalHom.lean

```diff
@@ -281,44 +326,70 @@ theorem coe_one : ((1 : A →ₙₐ[R] A) : A → A) = id :=
   rfl
 #align non_unital_alg_hom.coe_one NonUnitalAlgHom.coe_one
 
-theorem zero_apply (a : A) : (0 : A →ₙₐ[R] B) a = 0 :=
+theorem zero_apply (a : A) : (0 : A →ₛₙₐ[φ] B) a = 0 :=
   rfl
 #align non_unital_alg_hom.zero_apply NonUnitalAlgHom.zero_apply
 
 theorem one_apply (a : A) : (1 : A →ₙₐ[R] A) a = a :=
   rfl
 #align non_unital_alg_hom.one_apply NonUnitalAlgHom.one_apply
 
-instance : Inhabited (A →ₙₐ[R] B) :=
+instance : Inhabited (A →ₛₙₐ[φ] B) :=
   ⟨0⟩
 
+variable {φ' : S →* R} {ψ : S →* T} {χ : R →* T}
+
+set_option linter.unusedVariables false in
```

Comment:
Why is this now needed?

Reply:
Because of kappa, I believe, which is used implicitly in `comp`.